addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2233,6 +2233,7 @@ var SnippetsMenu = Widget.extend({
      */
     _computeSelectorFunctions: function (selector, exclude, target, noCheck, isChildren, excludeParent) {
         var self = this;
+        const dataExclude = exclude;
```

Comment:
It's a half-truth here, the exclude that arrives as a parameter has already been modified by `_computeSnippetTemplates` but since I can't modify the signature of the `_computeSelectorFunctions` function to add an "originalExclude" parameter in stable, I hope it's OK like that.